src/super.h

```diff
@@ -94,6 +96,13 @@ static inline bool SCOUTFS_HAS_SBI(struct super_block *sb)
 	return (sb != NULL) && (SCOUTFS_SB(sb) != NULL);
 }
 
+static inline bool SCOUTFS_IS_META_BDEV(struct scoutfs_super_block *super_block)
+{
+	return super_block->flags & SCOUTFS_FLAG_IS_META_BDEV;
```

Comment:
I'm never sure if we still need to worry about overflowing masking to bool.  It used to be faked with an int so bool() {return u64 & HIGH_BITS} would overflow to zero.  I still do !!(a & mask) in the code in places to turn conditions into strict 1 and 0.